client/beefy/src/worker.rs

```diff
@@ -524,20 +529,23 @@ where
 		match self.voting_oracle().triage_round(block_num, best_grandpa)? {
 			RoundAction::Process => {
 				debug!(target: LOG_TARGET, "🥩 Process justification for round: {:?}.", block_num);
+				metric_inc!(self, beefy_imported_justifications);
 				self.finalize(justification)?
 			},
 			RoundAction::Enqueue => {
 				debug!(target: LOG_TARGET, "🥩 Buffer justification for round: {:?}.", block_num);
 				if self.pending_justifications.len() < MAX_BUFFERED_JUSTIFICATIONS {
 					self.pending_justifications.entry(block_num).or_insert(justification);
+					metric_inc!(self, beefy_buffered_justifications);
```

Comment:
Is `beefy_buffered_justifications` supposed to be equal to `self.pending_justifications` ? I see that at decrementing we just do `metric_set!(self, beefy_buffered_justifications, self.pending_justifications.len());` . Would it make sense to do the same thing here ?

Reply:
yes, it will be `len()`, but it is cheaper to `.inc()` than do `.set(pending_justifications.len())`